helm-chart/kuberay-operator/values.yaml

```diff
@@ -74,6 +74,10 @@ securityContext:
     type: RuntimeDefault
 
 
+# enabled by default, disable leader election to solve k8s api timeout when leader election occur and 1 replica exist.
+leaderElection:
```

Comment:
Will there be any other options under `leaderElection`? If not, maybe we can use `leaderElectionEnabled` instead.

Reply:
Thank for the suggestion,
@kevin85421 seems like we hit API server limit, the question is why?

Reply:
should I have more than 1 replicas in kuberay? if so, I don't want to disable leader election.